packages/suite/src/hooks/wallet/useCoinmarketBuyOffers.ts

```diff
@@ -97,7 +97,7 @@ export const useOffers = (props: Props) => {
                 if (!quote.quoteId) {
                     const response = await invityAPI.doBuyTrade({
                         trade: quote,
-                        returnUrl: createQuoteLink(quotesRequest, account),
+                        returnUrl: await createQuoteLink(quotesRequest, account),
```

Comment:
declare `returnUrl` as a const before the invityAPI call.

async in async looks suspicious